components/CalEPD/include/dke/depg750bn.h

```diff
@@ -0,0 +1,74 @@
+// DKE product https://dke.top/products/dke-e-ink-103-color-displays-large-screen-display-102-inch-resolution-is-960x640
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "freertos/FreeRTOS.h"
+#include "esp_system.h"
+#include <stdint.h>
+#include <math.h>
+#include "sdkconfig.h"
+#include "esp_log.h"
+#include <string>
+#include <epd.h>
+#include <Adafruit_GFX.h>
+#include <epdspi.h>
+#include <gdew_4grays.h>
+#include <esp_timer.h>
+
+// Controller: SSD1681 (Acts similar than IL3829)
+#define DEPG750BN_WIDTH 800
+#define DEPG750BN_HEIGHT 480
+#define DEPG750BN_BUFFER_SIZE DEPG750BN_WIDTH * DEPG750BN_HEIGHT/8
+// 1 byte of this color in the buffer
+#define DEPG750BN_8PIX_BLACK 0xFF
+#define DEPG750BN_8PIX_WHITE 0x00
+
+class Depg750bn : public Epd
+{
+  public:
+	Depg750bn(EpdSpi& IO);
+
+    const uint8_t colors_supported = 1;
+    const uint8_t partial_supported = 1;
```

Comment:
Sorry though it had color because of the link. If it supports partial and it does work then it can be 1

Reply:
The display only supports black and white. No other color. and supports partial area update. Window update functionality has been added.